go.mod

```diff
@@ -6,6 +6,7 @@ toolchain go1.21.3
 
 replace (
 	github.com/cometbft/cometbft => github.com/oasisprotocol/cometbft v0.37.2-oasis1
+	github.com/oasisprotocol/oasis-sdk/client-sdk/go => /home/mitjat/oasis/oasis-sdk/client-sdk/go
```

Comment:
PR requires https://github.com/oasisprotocol/oasis-sdk/pull/1638/files and a new SDK release, then this override can go away. In a rush, we can hardcode this part of the SDK config into nexus temporarily.